src/workerd/io/container-client-test.wd-test

```diff
@@ -0,0 +1,24 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "container-client-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "container-client-test.js")
+        ],
+        compatibilityDate = "2025-01-09",
+        compatibilityFlags = ["nodejs_compat"],
+        docker = (address = "/var/run/docker.sock"),
+        durableObjectNamespaces = [
+          ( className = "DurableObjectExample",
+            uniqueKey = "210bd0cbd803ef7883a1ee9d86cce06e",
+            docker = (containerName = "docker-example") ),
```

Comment:
Would definitely want @kentonv's opinion on this structure. Would it actually make more sense to define docker as a kind of service appearing in the `services` collection? ... which would move it into the list alongside the `name = "internet"` and `name = "TEST_TMPDIR"` items below... then then linking this worker to the container can just be a service reference. 

Also, I'm not sure we should (or can) use the name `docker` in here. Docker is a trademarked name and its probably a bad idea to tie ourselves to it. Something generic like `container` is likely better in here.